install/kubernetes/cilium/templates/clustermesh-config/clustermesh-secret.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
   {{- end }}
 data:
   {{- $kvstoremesh := and .Values.clustermesh.useAPIServer .Values.clustermesh.apiserver.kvstoremesh.enabled }}
-  {{- $override := ternary (printf "https://clustermesh-apiserver.%s.svc:2379" .Release.Namespace) "" $kvstoremesh }}
+  {{- $override := ternary (join "\n- " $.Values.etcd.endpoints) (ternary (printf "https://clustermesh-apiserver.%s.svc:2379" .Release.Namespace) "" $kvstoremesh) .Values.clustermesh.kvstoremeshExternalKVStore }}
```

Comment:
When external KVStore mode is enabled, the Cilium agents appear to default using the etcd endpoints configured via `.Values.etcd.endpoints`. However, this change does not seem to account for the handling of the TLS certificates, accounting for `.Values.etcd.ssl`. Similarly, neither the endpoints nor the certificates seem to be automatically configured in the `kvstoremesh` container, which would not work out of the box.